src/Host/Broker/Impl/Sessions/SessionManager.cs

```diff
@@ -68,7 +69,8 @@ public class SessionManager {
                 password,
                 profilePath,
                 _loggingOptions.LogHostOutput ? _hostOutputLogger : null,
-                _loggingOptions.LogPackets ? _messageLogger : null);
+                _loggingOptions.LogPackets ? _messageLogger : null,
+                _loggingOptions.LogPackets ? LogVerbosity.Traffic : LogVerbosity.Minimal);
```

Comment:
`LogHostOutput` is for host process stderr. This includes the packets, which the host logs, but it shows host's view of what it gets.

`LogPackets` is for the packets as they flow through the broker. 

So yes, these two are completely independent, but both involve traffic.